### PR TITLE
Fletching fixes

### PR DESCRIPTION
**Pull request by EthanVieira**

### CODE REVIEW

**Comment by EthanVieira** [comment]
Just need the last commit checked

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## Play online
 
-#### [>Click here to play<](https://ethanvieira.github.io/rs-clicker/)
+#### [>Click here to play<](https://curtisgreen.github.io/rs-clicker/)
```
Comment (CurtisGreen) [inline README.md]: Need to keep this on yours

**Review by CurtisGreen — CHANGES_REQUESTED** [review]
Everything but the readme link looks good so far. I haven't merged a fork in before, should I close my PR or will that mess things up?

**Comment by EthanVieira** [comment]
> Everything but the readme link looks good so far. I haven't merged a fork in before, should I close my PR or will that mess things up?

Not sure. I have my own local copy of your forked repo pointing to this as its origin instead. I think the two branches/PRs are unique.

**Review by CurtisGreen — APPROVED** [review]
Alright looks good then, I'll close the PR on my fork as well

**Pull request merged**